.circleci/config.yml

```diff
@@ -1,77 +0,0 @@
-version: 2
```

Comment:
Deleting the remaining circle config causes the Circle CI build to fail with "No `.circleci/config.yml` was found in your project. Add a configuration file to your repository or refer to documentation.". When we get to merging this I will disable Circle from trying to run on the repository but for now I am going to leave it while there are still other open PRs running the service tests on Circle. Also I will update the branch protection rules, delete all the env vars from Circle, etc.